README.md

```diff
@@ -13,6 +13,7 @@ Available addons
 addon | version | summary
 --- | --- | ---
 [l10n_fi_banks](l10n_fi_banks/) | 10.0.1.0.0 | Finnish banks and their addresses
+[l10n_fi_edicode](l10n_fi_edicode/) | 10.0.1.1.2 | Adds EDI code field and operators
```

Comment:
Maybe add `l10n_fi_business_code` and `l10n_fi_business_code_validate` here too.

Reply:
I'll take that back. This section is autogenerated: https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md

Reply:
Maybe remove this modification (edicode) and let OCA bot autogenerate it.

Reply:
Done. The auto-generation was the reason why I didn't add the business code -modules in readme.